src/main/java/frc/robot/subsystems/DriveSubsystem.java

```diff
@@ -42,7 +47,8 @@ public DriveSubsystem() {
 		rightSpeedControllerGroup = new SpeedControllerGroup(frontRightMotor, backRightMotor);
 		
 		differentialDrive = new DifferentialDrive(leftSpeedControllerGroup, rightSpeedControllerGroup); 
-		
+		gyro = new AHRS(SPI.Port.kMXP);
+
```

Comment:
We should also reset the Gyro and make sure calibration finished before continuing.